wiki/Glossary/en.md

```diff
@@ -168,59 +171,59 @@ On the contrary, a reverse choke is when the player combobreaks at the beginning
    - when collaborating on a beatmap, the creator must map a majority (at least half) of the beatmap
    - this is to ensure that the creator has done enough to upload the map under their username
-2. the mapset containing guest difficulties
+2. the mapset containing [guest difficulties](#guest-difficulty)
 
-Collab may sometimes be used in difficulty version names.
+Collab may sometimes be used in [difficulty](/wiki/difficulty) version names.
 
 ### Combo
 
 Combo has two meanings:
 
 1. a set of hit circles and sliders that comes one after another (also called a _comboset_)
-2. the number representing the number of objects a player has hit consecutively without missing or sliderbreaking once
+2. the number representing the number of objects a player has hit consecutively without missing or [sliderbreaking](#sliderbreak) once
    - this number is displayed in various places:
-     - bottom-left in osu!standard
-     - on the drums in osu!taiko
-     - above the fruit catcher in osu!catch
-     - centre of the stage in osu!mania (vertical height varies by skin)
+     - bottom-left in [osu!standard](/wiki/osu!standard)
+     - on the drums in [osu!taiko](/wiki/osu!taiko)
+     - above the fruit catcher in [osu!catch](/wiki/osu!catch)
+     - centre of the stage in [osu!mania](/wiki/osu!mania) (vertical height varies by skin)
 
 A _max combo_ is simply the player's highest achieved combo throughout the play.
 
 ### Combo Colour
 
 The combo colour is a colour that that separates a comboset from another.
 
-In osu!standard, all combosets have a colour.
+In osu!standard, all [combosets](#comboset) have a colour.
 A new combo is started when the colour of the hit object changes.
-The colours will keep repeating themselves depending on how many the mapper defined (or how many your skin's skin.ini file defines).
+The colours will keep repeating themselves depending on how many the mapper defined (or how many your skin's `skin.ini` file defines).
 There is a minimum of 2 and a maximum of 8 combo colours.
 
 In osu!catch, each fruit is coloured in the order given.
 Droplets and end-slider fruit will always have the same colour as the start-fruit.
 However, bananas will always have a tint of yellow.
 
-When editing a beatmap, one can change those colours and/or their order in the Song Setup section.
+When editing a beatmap, one can change those colours and/or their order in the [Song Setup](/wiki/Song_Setup) section.
 
-For skins, the skin properties for the combo colours is located on in the `[Colours]` section of the skin.ini.
+For skins, the skin properties for the combo colours is located on in the `[Colours]` section of the `skin.ini`.
 However, do note that the skin's defined combo colours is only used when the mapper does **not** define them.
 
 ### Combo Fire
 
 ![](/wiki/shared/combo-fire.jpg "R.I.P. combo fire")
```

Comment:
using
``![Ancient screenshot of the combo fire](/wiki/shared/combo-fire.jpg "R.I.P. combo fire")``